apps/strapi-cms/.env.example

```diff
@@ -4,4 +4,26 @@ APP_KEYS=changeme1,changeme2
 API_TOKEN_SALT=changeme
 ADMIN_JWT_SECRET=changeme
 TRANSFER_TOKEN_SALT=changeme
-JWT_SECRET=changeme
\ No newline at end of file
+JWT_SECRET=changeme
+
+#DB CONFIG
+DATABASE_CLIENT=
```

Comment:
```suggestion
DATABASE_CLIENT=sqlite
```

Reply:
it's ok to set `sqlite` in the example, it wouldn't require any other operation when running the app

Reply:
I initially chose not to do this to avoid any remote risk of running a production instance and forgetting to set PostgreSQL as the database

I'll commit the suggestion ASAP if we're confident that won't happen.

Reply:
I'm pretty confident that we are not going to copy the `.env.default` values to the production instance 😄